### PR TITLE
Update phptaskman/core to ^1

### PR DESCRIPTION
**Pull request by ghost**
I wasn't able to install phptaskman/github together with other phptaskman packages because they were requiring ^1. Seems a good idea to follow suit with phptaskman/github to be able to use all packages together.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
         "ext-fileinfo": "*",
         "knplabs/github-api": "^2.11",
         "php-http/guzzle6-adapter": "^1.1",
-        "phptaskman/core": "^0.1.16"
```
Comment (drupol) [inline composer.json:17]: Hi mate,

The branch 0.x of Taskman is supporting PHP >= 5.6.
The branch 1.x of Taskman is supporting PHP >= 7.1.

There is currently no version of this package for Taskman 1.x. I think the best would be to merge this in master and create a branch 0.x from the previous commit.

What do you think ?

Comment (drupol) [reply composer.json:17]: I'll merge this and do the branch 0.x right now.

Comment (ghost) [reply composer.json:17]: That sounds good. I think with PHP 5.6 end of life the branches 0.x will not need much more work anyway. But good to have the branch there.

Comment (drupol) [reply composer.json:17]: It's done! Release 1.0.0 of this has been tagged as well :-)

**Pull request merged**